ports/libunifex/vcpkg.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "libunifex",
+  "version-date": "2021-08-01",
+  "description": "Unified Executors",
+  "homepage": "https://github.com/facebookexperimental/libunifex",
+  "supports": "linux | windows",
```

```suggestion
  "supports": "!osx",
```
99% of the time, we prefer to be open, not closed (unless the code literally fatal errors on `system_name` other than exactly: Linux, Windows, WindowsStore